hudi-common/src/main/java/org/apache/hudi/common/table/view/FileSystemViewStorageConfig.java

```diff
@@ -52,7 +54,7 @@
   public static final String DEFAULT_FILESYSTEM_VIEW_INCREMENTAL_SYNC_MODE = "false";
   public static final String DEFUALT_REMOTE_VIEW_SERVER_HOST = "localhost";
   public static final Integer DEFAULT_REMOTE_VIEW_SERVER_PORT = 26754;
-
+  public static final Integer DEFAULT_REMOTE_TIMELINE_CLIENT_TIMEOUT_SECS = 300 * 60; // 5 min
```

Comment:
@prashantwason How is this 5 mins ? If it's secs, it should just be 300 ?

Reply:
Corrected.